plugins/urbandictionary/urbandictionary.js

```diff
@@ -0,0 +1,52 @@
+const https = require('https');
+const Bot = require('../../modules/Bot.js');
+
+
+module.exports = {
+  name: 'Urban Dictionary',
+  description: 'slang dictionary and for some meme :)',
+  author: "Created by Rehkloos",
+  license: "Apache-2.0",
+  command: 'ud', // This is the Command that is typed into Chat!
+  permissions: [], // This is for Permissisons depending on the Platform.
+  cooldown: 10, // this is Set in Seconds, how long between the next usage of this command.
+  execute(client, data, modules) {
+    const searchTerm = data.args[0];
+
+    let url = `https://api.urbandictionary.com/v0/define?term=${searchTerm}`;
+
+    https.get(url, (res) => {
```

Comment:
y u not use Tool.js @.@

Reply:
any json requests with an array causes tools.js functions to crap out on me, if its a simple json request then it works fine